src/imu/imu.h

```diff
@@ -18,13 +18,14 @@
 *
 * The purpose of this test is to evaluate the accuracy of the IMU Euler angles measurements.
 * It takes as input the urdf of the robot and make a comparison between the expected values retrieved from the forward kinematics and the ones read from the IMU itself.
-* The test involves the movements of the joints belonging to the part on which the sensor is mounted. The movements are executed sequentially, traversing from the home position to the lower limit, upper limit and back to the home position for each joint.
+* The test involves the movements of the joints belonging to the part on which the sensors are mounted.
```

Comment:
Looking at the configuration file https://github.com/robotology/icub-tests/blob/devel/suites/contexts/icub/test_imu.ini#L3, it seems that the test requires the existence of the `multipleanalogsensorsserver` with prefix `/${robotname}/alljoints/inertials` . Which robots expose this functionality? It could make sense to either list the robots that expose this functionality, or if that is too complicated from the maintenance point of view, just explicitly mention that the robot needs to expose a `multipleanalogsensorsserver` device with prefix `/${robotname}/alljoints/inertials` in its `yarprobotinterface` config files.

Reply:
So far, no robots exposed this functionality upstream but I'm going to do this at least for `iCubGenova11` and for `iCubGazeboV2_*` models I guess. We could use them as references to highlight in the documentation that is mandatory to have this MASserver exposed with that port name. Thanks @traversaro!